egs/sprakbanken/s5/run.sh

```diff
@@ -125,19 +124,43 @@ if [ $stage -le 9 ]; then
 fi
 
 if [ $stage -le 10 ]; then
-# Alignment used to train nnets and sgmms
-steps/align_fmllr.sh --nj $nj --cmd "$train_cmd" \
-  data/train data/lang exp/tri3b exp/tri3b_ali || exit 1;
+  # Alignment used to train nnets and sgmms
+  steps/align_fmllr.sh --nj $nj --cmd "$train_cmd" \
+    data/train data/lang exp/tri3b exp/tri3b_ali || exit 1;
 fi
 
-##TODO: Add nnet3 and chain setups
 
 ## Works
 #local/sprak_run_nnet_cpu.sh tg dev 
 
 ## Works
 #local/sprak_run_sgmm2.sh dev
 
+# Run neural network setups based in the TEDLIUM recipe
+if [ $stage -le 11 ]; then
```

Comment:
Hm.  I'm not sure.  I'd probably prefer to include the command (or commands) but commented out, with a comment saying that this is the command to run [such-and-such.]